assets/src/application/ui/input/EspressoButton/Tooltip.tsx

```diff
@@ -27,10 +26,11 @@ const Tooltip: React.FC<TicketsChainedButtonProps> = ({ button, buttonText, isMo
 	return isMobile ? (
 		mobileHelpText
 	) : (
-		<DefaultTooltip title={tooltip} {...props}>
+		<DefaultTooltip title={title} {...props}>
```

Comment:
why this change? what was wrong with the prop being named "tooltip"?
we don't need to use the same prop names as the ANtD components!

Reply:
Actually when I disabled that viewpoint hook, it raised TS error that `title` prop is required for AntD Tooltip

Reply:
that's bonkers:

```js
<DefaultTooltip title={tooltip} {...props}>
```